csvcubed/csvcubed/readers/configdeserialiser.py

```diff
@@ -122,7 +122,7 @@ def _metadata_from_dict(config: dict) -> "CatalogMetadata":
 
     return CatalogMetadata(
         identifier=config.get("id"),
-        title=config.get("title"),
+        title=config.get("title", ""),
```

Comment:
I'd say we should be raising an exception if the user hasn't provided a title. It's a required value and I really don't like it being left empty. It'll make it almost impossible to find the dataset when uploaded to PMD.